plat/hisilicon/hikey/platform.mk

```diff
@@ -82,3 +93,9 @@ BL31_SOURCES		+=	plat/hisilicon/hikey/hisi_sip_svc.c			\
 				lib/pmf/pmf_smc.c
 endif
 
+# Enable workarounds for selected Cortex-A53 errata.
+ERRATA_A53_835769		:=	1
+ERRATA_A53_843419		:=	1
+ERRATA_A53_855873		:=	1
+ERRATA_A53_826319		:=	1
+ERRATA_A53_836870		:=	1
```

Comment:
Enabling these workarounds is off-topic and should be done in a separate commit. Some of them are also enabled in https://github.com/ARM-software/arm-trusted-firmware/pull/1013. Please agree with @hzhuang1 on which ones to enable.